### DISCUSSION TITLE
How do i disable the CodeQL scanning after i have enabled it by adding a workflow?

### DISCUSSION THREAD

**Discussion by ashraf-vazeer** [community/community#39134] [Code Security] [answered]
### Select Topic Area

Question

### Body

Hi,

I have added workflow to scan the code using CodeQL.
Now I want to disable it.

Is it enough to just remove the workflow .github/workflows/codeql.yml ?

Thanks
Ashraf

**Comment by aeisenberg** [comment, marked as answer]
Hello, If you delete the workflow file, the alerts that it found will still be marked as opened. If there are not many alerts, I would suggest manually closing everything. If there are lots of alerts, too many to close manually, then you can use the github api to delete analysis runs. See https://docs.github.com/en/rest/code-scanning#delete-a-code-scanning-analysis-from-a-repository.

**Reply by ashraf-vazeer** [reply]
Thankyou will try this out and get back in case any issues

**Reply by Dragon1573** [reply]
Thank you ❤️ 

Now I reset all my CodeQL configurations with running the following command.

```powershell
gh api '/repos/Dragon1573/<repo>/code-scanning/analyses' --jq '.[].id' |
Sort-Object -Descending |
ForEach-Object {
  gh api --method DELETE "/repos/Dragon1573/<repo>/code-scanning/analyses/${_}?confirm_delete=true"
}
